tests/test_workers/test_config.py

```diff
@@ -291,35 +292,23 @@ def test_validate_celery_config_invalid_s3_env_vars():
         validate_celery_config(conf)
 
 
-@pytest.mark.parametrize(
-    'config, error',
-    (
-        (
-            {'iib_otel_tracing': True},
-            (
-                '"OTEL_EXPORTER_OTLP_ENDPOINT" and "OTEL_SERVICE_NAME" environment '
-                'variables must be set to valid strings when iib_otel_tracing is set to True.'
-            ),
-        ),
-        (
-            {'iib_otel_tracing': 'random-str'},
```

Comment:
I know that nobody likes to write tests.
However it would be nice also cover the case when `IIB_OTEL_TRACING` is set incorrectly or not set at all (as it was in this test before).

`@mock.patch.dict(os.environ, {'IIB_OTEL_TRACING': 'random-str'}) `

Reply:
Aha, that case is indeed absence of failure. I removed it because if you specify anything other than 'True', it should evaluate as false and not check otel variables. I'll add it to a test that validates successfully. Would that suffice?

Reply:
Added. Please let me know what you think